src/ClientModel/SamplingDataPostProcessor.cpp

```diff
@@ -125,13 +132,16 @@ class SamplingDataPostProcessor {
 
 PostProcessedSamplingData CreatePostProcessedSamplingData(const CallstackData& callstack_data,
                                                           const CaptureData& capture_data,
+                                                          const ModuleManager* module_manager,
```

Comment:
Why is `module_manager` passed by pointer while `callstack_data` and `capture_data` by reference? Can `module_manager` be `nullptr`? I thought we always have a `ModuleManager`. This applies also to `SamplingDataPostProcessor` itself.